frontend/apps/cloudstorage/dropbox.lua

```diff
@@ -92,17 +92,18 @@ function DropBox:createFolder(url, password, folder_name, callback_close)
 end
 
 function DropBox:config(item, callback)
-    local text_info = "How to generate Access Token:\n"..
+    local text_info = "How to link your Dropbox account:\n"..
```

Comment:
Sorry, I'd overlooked this untranslated string earlier but if you don't mind (since  it's  not really  part of this PR per se), could you make this:

_([[multiline string]])

That should be the easiest for translators.

Reply:
I think its done, let me know if I should change anything else